workspace/manager.py

```
@@ -240,12 +253,165 @@ def create_workspace(
+            if token and repo_url.startswith("https://"):
+                auth_url = repo_url.replace("https://", f"https://{token}@")
+            proc = await asyncio.wait_for(
+                asyncio.create_subprocess_exec(
+                    "git", "ls-remote", "--heads", auth_url,
+                    stdout=asyncio.subprocess.PIPE,
+                    stderr=asyncio.subprocess.PIPE,
+                    env=env,
+                ),
+                timeout=timeout,
+            )
+            stdout, stderr = await proc.communicate()
```

Ruff (0.15.12)

[warning] Unpacked variable `stdout` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)